src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -231,85 +232,72 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
 
     return (
       <>
-        <Row>
-          <Col>
-            <BuyNowStepper currentStep={"shipping"} />
-          </Col>
-        </Row>
+        <HorizontalPadding size={4} sizeXS={0}>
```

Comment:
Another thought would be to piggy back on styled-system's [responsive props](https://github.com/jxnblk/styled-system/blob/master/docs/responsive-styles.md). So like 

```
<HorizontalPadding p={[0, 4]}>